requirements.txt

```diff
@@ -1,6 +1,7 @@
 # file generated by https://github.com/bndr/pipreqs
 # Version updates managed by pyup.io
 
+attrdict==2.0.1
```

Comment:
any reason for the new dependency instead of just making a basic AttrDict class that can do the same?
not a blocker

Reply:
Because I don't want to effectively re-implement all of this:
https://github.com/bcj/AttrDict/blob/master/attrdict/mixins.py